### PR TITLE
A blog post about images updates

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

**Comment by sboldyreva** [comment]
@bennyvasquez

**Comment by codyro** [comment]
https://efada830.almalinux-org.pages.dev/blog/2023-12-21-almalinux-container-cloud-live-media-updates/

LGTM

**Review by codyro — APPROVED** [review]

File: content/blog/2023-12-21-almalinux-container-cloud-live-media-updates.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "AlmaLinux Cloud, Container and Live Media Images Updates"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: 2023-12-21
```
Comment (bennyvasquez) [inline content/blog/2023-12-21-almalinux-container-cloud-live-media-updates.md]:
```suggestion
date: 2023-12-26
```

**Review by bennyvasquez — APPROVED** [review]

**Comment by bennyvasquez** [comment]
https://5f655087.almalinux-org.pages.dev/

File: content/blog/2023-12-26-almalinux-container-cloud-live-media-updates.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Cloud and Container Images Updates and Changes
+
+AlmaLinux OS provides a variety of cloud images, so we've been updating them to 9.3 and 8.9 versions including: 
+* [Amazon AWS](https://wiki.almalinux.org/cloud/AWS.html)
+    * Starting AlmaLinux OS versions 8.9 and 9.3 Amazon Machine images have UEFI boot support. Thus boot mode was changed to the `uefi-preferred` meaning that UEFI will be chosen on the supported instance types. We are planning to publish separate AMIs with enabled Secure Boot and NitroTPM(vTPM) support.
+* [Generic Cloud](https://wiki.almalinux.org/cloud/Generic-cloud.html)
+* [OpenNebula](https://wiki.almalinux.org/cloud/OpenNebula.html)
+    * Starting AlmaLinux OS 8.9 Generic Cloud and OpenNebula images have hybrid - BIOS and UEFI - boot support. The download URLs of the UEFI images are symlinked to the current image for compatibility.
+* [Microsoft Azure](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64)
```
Comment (LKHN) [inline content/blog/2023-12-26-almalinux-container-cloud-live-media-updates.md:27]: Can we add the AArch64 version of Azure offer too? 
https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-arm

File: content/blog/2023-12-26-almalinux-container-cloud-live-media-updates.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Hey Community! We're just sharing some news on both AlmaLinux 9.3 and 8.9 images updates. 
+
+
+## Cloud and Container Images Updates and Changes
+
+AlmaLinux OS provides a variety of cloud images, so we've been updating them to 9.3 and 8.9 versions including: 
+* [Amazon AWS](https://wiki.almalinux.org/cloud/AWS.html)
+    * Starting AlmaLinux OS versions 8.9 and 9.3 Amazon Machine images have UEFI boot support. Thus boot mode was changed to the `uefi-preferred` meaning that UEFI will be chosen on the supported instance types. We are planning to publish separate AMIs with enabled Secure Boot and NitroTPM(vTPM) support.
+* [Generic Cloud](https://wiki.almalinux.org/cloud/Generic-cloud.html)
```
Comment (LKHN) [inline content/blog/2023-12-26-almalinux-container-cloud-live-media-updates.md:24]: It's only applies to the Generic Cloud images, not the OpenNebula ones.

File: content/blog/2023-12-26-almalinux-container-cloud-live-media-updates.md (hunk 2 above)
Comment (sboldyreva) [reply content/blog/2023-12-26-almalinux-container-cloud-live-media-updates.md]: @LKHN please check

**Comment by LKHN** [comment]
LGTM

**Pull request merged**